lib/Cavil/Model/Snippets.pm

```diff
@@ -35,4 +35,14 @@ sub find_or_create {
   return $db->select('snippets', 'id', {hash => $hash})->hash->{id};
 }
 
+sub random {
+  my ($self, $limit) = @_;
+
+  return $self->pg->db->query(
+    'select id, text, classified,
+         license, confidence from snippets where approved=FALSE
+        order by random() limit ?', $limit
```

Comment:
this seems not to work performant for larger tables.

Reply:
This is faster, but you need to know the number of rows in the table. 🤔
```sql
select id, text, classified, license, confidence from snippets where approved=false offset floor(random()*427040) limit 100;
```

Reply:
Luckily getting the highest id is a very fast query (unlike count).
```sql
select max(id) from snippets;
```

Reply:
then we better have no gaps in the ids.

Reply:
Getting the estimate is also very fast.
```sql
select reltuples as approximate_row_count from pg_class where relname = 'snippets';
```

Reply:
ordering by (indexed) hash proved to be fast and random enough :)